SimpleRss/SimpleRss/Views/ViewExtensions/ImageViewExtensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ImageViewExtensions.swift
+//  SimpleRss
+//
+//  Created by Voldem on 12/13/18.
+//  Copyright © 2018 Vladimir Koptev. All rights reserved.
+//
+
+import UIKit
+
+extension UIImageView {
```

Comment:
This extension doesn't seem to be used anywhere.

Reply:
It's used in HomeViewController        
cell.previewImageView.downloaded(from: topics[indexPath.row].picUrl)